wagtailnhsukfrontend/static/wagtailnhsukfrontend/css/fixes.css

```diff
@@ -2,6 +2,5 @@
 
 /* wagtail RichText fields and blocks add a .rich-text div around content */
 div.rich-text :last-child {
-  margin-bottom: 0;
```

Comment:
this file doesn't need any changes

Reply:
Resolved, not sure why that changed, must be my VS formatting.